src/app/services/display.service.ts

```diff
@@ -25,87 +26,95 @@ export class DisplayService implements OnDestroy {
         return this._currentRun$.asObservable();
     }
 
-    public get currentRun(): Run {
```

Comment:
Der richtige Wert (anstatt des Observables) sollte man nur in Ausnahmesituationen verwenden. Sonst werden Änderungen am Objekt nicht mitbekommen